examples/enterprise_edr/device_processes.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+import sys
+import logging
+import concurrent.futures
+from cbc_sdk.helpers import eprint, build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import Device
+from cbc_sdk.enterprise_edr import Process
+
+log = logging.getLogger(__name__)
+
+
+def main():
```

Comment:
Can we add some comments around the usage pattern for the cli input

Reply:
Why? None of our other examples has those.

Reply:
This script does not have any descriptions of what it does we need at a minimum some context for what this does so a developer can't better utilize the example. Our other examples should have descriptions too

Reply:
I added a docstring to the script to explain what exactly it does.  I suggest that an additional story be created to add those to other examples.